WooCommerce/Classes/ViewRelated/Orders/Fulfillment/Cells/PickListTableViewCell.swift

```diff
@@ -0,0 +1,99 @@
+import UIKit
+import Gridicons
+import Yosemite
+
+
+/// Pick List: Renders a row that displays a single Product.
+///
+class PickListTableViewCell: UITableViewCell {
+
+    /// ImageView
+    ///
+    @IBOutlet private var productImageView: UIImageView!
+
+    /// Label: Name
+    ///
+    @IBOutlet private var nameLabel: UILabel!
+
+    /// Label: Quantity
+    ///
+    @IBOutlet private var quantityLabel: UILabel!
+
+    /// Label: SKU
+    ///
+    @IBOutlet private var skuLabel: UILabel!
+
+    /// Product Name
+    ///
+    var name: String? {
+        get {
+            return nameLabel?.text
+        }
+        set {
+            nameLabel?.text = newValue
+        }
+    }
+
+    /// Number of Items
+    ///
+    var quantity: String? {
+        get {
+            return quantityLabel?.text
+        }
+        set {
+            quantityLabel?.text = newValue
+        }
+    }
+
+    /// Item's SKU
+    ///
+    var sku: String? {
+        get {
+            return skuLabel?.text
+        }
+        set {
+            skuLabel?.text = newValue
+        }
+    }
+
+    // MARK: - Overridden Methods
+
+    required init?(coder aDecoder: NSCoder) {
+        // initializers don't call property observers,
+        // so don't set the default for mode here.
+        super.init(coder: aDecoder)
+    }
+
+    class func makeFromNib() -> PickListTableViewCell {
+        return Bundle.main.loadNibNamed("ProductDetailsTableViewCell", owner: self, options: nil)?.first as! PickListTableViewCell
+    }
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        productImageView.image = Gridicon.iconOfType(.product)
+        productImageView.tintColor = StyleManager.wooGreyBorder
+        selectionStyle = .none
+
+        nameLabel.applyBodyStyle()
```

Comment:
I find slightly more readable to encapsulate the initial setup for each outlet in a separate method, that way this would read as:

```
setupQuantityLabel()
setupNameLabel()
setupSKULabel()
```

and then on each of those methods, apply the style and initialise the value to an empty string. What do you think?

Reply:
Thanks for the reminder. I struggle with encapsulation if you can't tell :)